src/Neo/Cryptography/Base58.cs

```diff
@@ -74,19 +84,28 @@ public static byte[] Decode(string input)
             var bi = BigInteger.Zero;
             for (int i = 0; i < input.Length; i++)
             {
-                int digit = Alphabet.IndexOf(input[i]);
-                if (digit < 0)
+                var digit = s_mapBase58[(byte)input[i]];
```

Comment:
Change to char array and avoid cast?

Reply:
`Char` wouldn't work I don't think. `Char` uses `encoding`.